posthog/warehouse/types.py

```diff
@@ -8,6 +8,7 @@ class IncrementalFieldType(StrEnum):
     DateTime = "datetime"
     Date = "date"
     Timestamp = "timestamp"
+    String = "string"
```

Comment:
I'd add a comment to say how this is needed for ObjectIDs in Mongo - otherwise its not super clear that sources dont need to support string incremental fields

Reply:
Hmm.. im conflicted on this actually. Sure, the `_id` can be interpreted as a string, but we're actually using it as a timestamp and all other logic will be treating the value as a timestamp (e.g. in `process_incremental_value(...)`) - so on second thought, I dont think we should have this string type in here